sql/updates/world/world_2019_01_25_02.sql

```diff
@@ -0,0 +1 @@
+REPLACE INTO `trinity_string` (`entry`, `content_default`) VALUES (2026, 'Showing list of escalated tickets.');
```

Comment:
That specific line is okay, I didn't review the rest yet

Reply:
:'D